app/views/inventories/_form.html.erb

```diff
@@ -0,0 +1,78 @@
+<%= form_with model: inventory do |form| %>
+  <% if inventory.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(inventory.errors.count, "error") %> prohibited this friend from being saved:</h2>
+
+      <ul>
+        <% inventory.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field form-group">
+    <%= form.label :model_id, class: "form-label" %>
+    <%= form.grouped_collection_select :model_id, Brand.order(:name), :models, :name, :id, :name, autofocus: true, class: "form-control"%>
```

Comment:
It seems class 'form-control' doesn't work for model, year, status
![image](https://user-images.githubusercontent.com/48946963/125375978-36539d80-e3b4-11eb-9723-5b935dc54610.png)

Reply:
I've figured out how to fix it, you can check the effect again.